lib/matestack/ui/vue_js/components/toggle.js

```diff
@@ -7,7 +7,7 @@ const componentDef = {
   data: function(){
     return {
       showing: true,
-      hide_after_timeout: null,
+      hideAfterTimeout: null,
```

Comment:
No need to rename this from my POV, but if you want to then please do so in the rest of the file as well ;)